ractive-koa/server/services/config.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+// >> CONFIGURATION
+// >> include this file for access to (`.get/.set`) configuration variables
+
+var key;
+var nconf = require('nconf');
+var env = process.env.NODE_ENV ? process.env.NODE_ENV : 'development';
+var config = {
+	env: env
+};
+var configFile;
+
+// load commandline arguments and environment variables
+nconf.argv().env();
+
+try {
+	// load any additional configuration
+	// NOTE: as this runs only once at start up,
+	// it's fine to run synchronous `try/catch` code
+	configFile = require('./config/' + env);
+	for (key in configFile) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.